recipes/prov/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "1.4.0" %}
+
+package:
+  name: prov
+  version: {{ version }}
+
+source:
+  fn: prov-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/1d/bb/9fd5313a33d55fb02a5f885407d6025b7d6fa1ad1b65713107756edba4db/prov-{{ version }}.tar.gz
+  md5: 251576e384ab87eb68bee84697e863a3
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Please add the arguments ` --single-version-externally-managed --record record.txt`. We do this when using `setuptools` to indicate that we don't want it try to handle package management for us.